tests/vmi_lifecycle_test.go

```diff
@@ -1224,11 +1248,11 @@ var _ = Describe("[rfe_id:273][crit:high][arm64][vendor:cnv-qe@redhat.com][level
 				node := nodes.Items[0].Name
 
 				By("Creating a VirtualMachineInstance with different namespace")
-				vmi = libvmi.New(
-					libvmi.WithResourceMemory("1Mi"),
-					libvmi.WithNetwork(v1.DefaultPodNetwork()),
-					libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding()),
-				)
+				vmi = libvmi.New(options...)
+				if arch == "s390x" {
+					vmi = libvmifact.NewAlpine(options...)
```

Comment:
Again lets document the justification for this

Reply:
review comment not applicable after reverting the changes.